docs/pages/management/diagnostics/metrics.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Metrics
+description: How to enable and consume metrics
+---
+
```

Comment:
I think this could use 1-2 sentences here about the purpose of this guide. Maybe something like:

```markdown
Teleport exposes metrics for all of its components, helping you get insight 
into the state of your cluster. This guide explains the metrics that you can 
collect from your Teleport cluster.
```